app/assets/javascripts/ransack_ui_jquery/index.js

```diff
@@ -1,2 +1,6 @@
 //= require ransack/predicates
-//= require ransack_ui_jquery/search_form
\ No newline at end of file
+//= require ransack_ui_jquery/search_form
+
+if(jQuery.fn.select2) {
+  window.Select2 = jQuery.fn.select2;
+}
```

Comment:
Which way is the new one? I think we want to have it the other way so we don't have to change it everywhere.

I guess no need to check if its' already defined there, right? It's either in one place or the other, never both?